src/coreclr/src/tools/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -2850,6 +2850,7 @@ private uint getJitFlags(ref CORJIT_FLAGS flags, uint sizeInBytes)
             flags.Set(CorJitFlag.CORJIT_FLAG_USE_PINVOKE_HELPERS);
 
             TargetArchitecture targetArchitecture = _compilation.TypeSystemContext.Target.Architecture;
+            TargetOS targetOS = _compilation.TypeSystemContext.Target.OperatingSystem;
```

Comment:
Nit: Given that this is used just once and only on x86, it does not seem to be worth it to cache it in a local.